src/uk/gov/defra/ffc/Tests.groovy

```diff
@@ -36,15 +36,14 @@ class Tests implements Serializable {
       }
   }
 
-  static def runPa11y(ctx, projectName, buildNumber, tag) {
-    def pa11yDockerComposeFile = 'docker-compose.pa11y.yaml'
-      ctx.gitStatusWrapper(credentialsId: 'github-token', sha: Utils.getCommitSha(ctx), repo: Utils.getRepoName(ctx), gitHubContext: GitHubStatus.Pa11y.Context, description: GitHubStatus.Pa11y.Description) {
+  static def runAccessibility(ctx, projectName, buildNumber, tag, accessibilityTestType) {
+    def dockerComposeFile = "docker-compose." + accessibilityTestType + ".yaml"
```

Comment:
I'd use a template for this, rather than concat'ing string.

ie. `def dockerComposeFile = "docker-compose.${accessibilityTestType}.yaml"